resources/views/emails/sample_notification.blade.php

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Please change the name of this blade file to something more understandable, like, email notification.